rosetta-source/src/main/rosetta/product-collateral-func.rosetta

```diff
@@ -1,22 +1,165 @@
 namespace cdm.product.collateral
 version "${project.version}"
 
+import cdm.base.staticdata.asset.common.*
 
 func Create_EligibleCollateralSpecificationFromInstruction: <"Creates an Eligible Collateral Specification through the combination of common and variable characteristics.">
-
     inputs:
         instruction EligibleCollateralSpecificationInstruction (1..1)
     output:
         specification EligibleCollateralSpecification (1..1)
 
     add specification -> criteria:
         instruction -> variable
-            extract MergeEligibleCollateralCriteria(item, instruction->common)
+            extract MergeEligibleCollateralCriteria(item, instruction -> common)
 
 func MergeEligibleCollateralCriteria: <"Java implementation merges criteria1 and criteria2.">
-
     inputs:
         criteria1 EligibleCollateralCriteria (1..1)
         criteria2 EligibleCollateralCriteria (1..1)
     output:
-        criteria EligibleCollateralCriteria (1..1)
\ No newline at end of file
+        criteria EligibleCollateralCriteria (1..1)
+
+
+func CheckEligibility:
```

Comment:
I recommend enriching the name of the functions to avoid future clashes. e.g  CheckCollateralEligibility. For example, we may also have Eligibility check in DRR context.

Reply:
Hi @rvincentISDA  - I am closing this to PR as I will submit a new one with better names and documentation. I am working closely with the ISDA Collateral representative to make sure the implementation is good.

Thank you

Minesh